### PR TITLE
Week 3 assessment tc

### PR DESCRIPTION
**Pull request by tori-calkins**
# Submitting Week 3 Assessment

### ✅ Completed Files

Select the following as confirmation that the files in this repository are complete.

- [X] Coding Challenges
- [X] Coding Concepts
- [X] Interview Questions

### ✋ Reminder

Do NOT merge your assessment!

### 📝 Feedback

If you would like specific feedback please leave details here:

### CODE REVIEW

File: code-challenges.test.js [hunk 1]
```diff
@@ -23,51 +38,109 @@ const fibonacciLength2 = 10
 // b) Create the function that makes the test pass.
 
 // Pseudo code:
+// function name: fibonacci
+// input: number, the length of the expected array
+// output: an array with the values that follow the fibonacci rules
+// process: create an array of nothing to hold the new array at the end, create a function involving a for loop to iterate through the length and stop once it hits the length of 6 and 10, number will be the parameter,
```
Comment (chrisyhlee) [inline code-challenges.test.js:44]: You explain making an array, and how the for loop works, but did not explain the meat of the code, which is the logic within the for loop. Make sure your pseudo code explains how the code is working!

File: code-challenges.test.js [hunk 2]
```diff
@@ -23,51 +38,109 @@ const fibonacciLength2 = 10
...
+// output: an array of the object values in the key:value pairs in order of least to greatest
+// process: create a function, Object.values() to move the values of the key: value pairs of the object into an array, .sort() .toFixed() method 
+const weekArr = (obj) => {
+  const var1 = Object.values(obj)
+  const var2 = var1.sort((a, b) => a - b)
+  return var2
+}
+//console.log(weekArr(studyMinutesWeek2))
+
+//  this one took forever for me to get it to pass the jest test, but it worked. my object.values wasn't capitalized so it kept saying it wasnt defined.
```
Comment (chrisyhlee) [inline code-challenges.test.js:103]: Nice catch! In your pseudo code, you mentioned the .toFixed() method, but it's not used in the function. Make sure your pseudo code reflects your code!

**Review by chrisyhlee — APPROVED** [review]
Tori, you are able to look at code and get an idea of how it's working, but don't have a full understanding of the concepts and tools you're using. This indicates a Novice comprehension level. Your pseudo code is explaining the tools you're using, but not how the code is working. For challenge 3, you did try to explain how the code was working, but the explanation showed confusion on how things were working (for example, you said .map creates a new array with values that meet the condition, but .map does not have any conditionals built into it). My main suggestion for you is to work on making your pseudo code clear and accurate, and you will move up to Advanced Beginner in no time. You're doing great!

File: code-challenges.test.js (hunk 2 above)
Comment (tori-calkins) [reply code-challenges.test.js:103]: I need to work on remembering to update my pseudocode as I go.

**Pull request merged**